src/meta_api.cpp

```diff
@@ -107,6 +107,7 @@ C_DLLEXPORT int Meta_Detach(PLUG_LOADTIME now [[maybe_unused]], PL_UNLOAD_REASON
     gSPGlobal->getCommandManager()->clearCommands();
     gSPGlobal->getCvarManager()->clearCvars();
     gSPGlobal->getMenuManager()->clearMenus();
+    gSPGlobal->getMessageManager()->clearMessages();
     gSPGlobal->getNativeProxy()->clearNatives();
 
```

Comment:
@Amaroq7 why do we clean all these things manually? Can't we just construct `gSPGlobal` in `Meta_Attach` and destruct it in `Meta_Detach`? (probably using `new` and `delete`)

Reply:
This could be done, but I'd leave it for another refactoring as well.